actions/methods.go

```diff
@@ -42,7 +42,8 @@ func MethodNotAllowed(c *gin.Context) {
 	c.JSON(http.StatusMethodNotAllowed, dictionary.GetError(dictionary.ErrorMethodNotAllowed, c.Request.Method, c.Request.RequestURI))
 }
 
-// GetSearchQueryParameters get all filtering parameters related to the db query
+// GetSearchQueryParameters get all filtering parameters related to the db
+// TODO: return map not pointer to map
```

Comment:
Should it remain here?

Reply:
Removed. We probably should look at this, but it's out of scope